src/golang/objectprotection/rubrik_snappable_protection_status.go

```diff
@@ -0,0 +1,92 @@
+package objectprotection
+
+import (
+	"log"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/rubrikinc/rubrik-sdk-for-go/rubrikcdm"
+)
+
+var (
+	// Rubrik Snappable SlaDomain Information
+	snappableEffectiveSlaDomain = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "rubrik_snappable_effective_sla",
+			Help: "Return the slaDomain information for snappables",
+		},
+		[]string{
+			"clusterName",
+			"objectName",
+			"objectType",
+			"objectID",
+			"location",
+			"slaDomain",
+		},
+	)
+)
+
+func init() {
+	// VMware vSphere VM effective sla domain
+	prometheus.MustRegister(snappableEffectiveSlaDomain)
+}
+
+// GetSnappableEffectiveSlaDomain ...
+func GetSnappableEffectiveSlaDomain(rubrik *rubrikcdm.Credentials, clusterName string) {
+	reportData, err := rubrik.Get("internal", "/report?report_template=ObjectProtectionSummary&report_type=Canned", 60) // get our object protection summary report
+	if err != nil {
+		log.Printf("Error from objectprotection.GetVSphereVmEffectiveSlaDomain: ", err)
```

Comment:
Looks like a copy 'n paste related accident.

Reply:
I.e `objectprotection.GetVSphereVmEffectiveSlaDomain` in the log message.

Reply:
Well spotted. The function was initially called GetVSphereVmEffecitveSlaDomain, but I decided to target all snappables in one go rather than do them in turn, so renamed the function, but I seem to have forgotten to replace the sting messages as well. I'll fix and push a new commit. Thanks.